Soccar [UI]/src/nl/socnet/message/handler/PlayerLeftSessionMessageHandler.java

```diff
@@ -32,6 +35,11 @@ protected void handle(Connection connection, PlayerLeftSessionMessage message) t
         }
         
         team.leave(optional.get());
+        
+        Object controller = Main.getInstance().getController();
+        if (controller != null && controller instanceof SessionViewFXMLController) {
+            Platform.runLater(() -> ((SessionViewFXMLController) controller).setRoomInfo());

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1612)